### ISSUE TITLE
nsp@1.0.2 commands return no output

### ISSUE THREAD

**Issue by reedloden** [nodesecurity/nsp#74]
Upgraded from 1.0.1 to 1.0.2, and now no nsp commands are returning output.

```
| => nsp blah
Unknown command "blah". Did you mean:
- package
- audit-package
- shrinkwrap
- audit-shrinkwrap
- version
```

```
| => nsp version
(no output)
```

```
| => nsp package
(no output)
```

```
| => nsp shrinkwrap
(no output)
```

Possibly due to chalk upgrade in 32f047165cdf4b91345f973f047dc5128ecf369c ?

**Comment by nlf** [comment]
this was a result of deep dependency in the shrinkwrap having been bumped, fixed in e3391e65802191b7f81774495fde97f58a31267c

**Comment by tripu** [comment]
I'm using `1.0.3` and I had this same issue. I think it happened because I ran `npm update`, and after doing that, one of the two instances of `sift` under my `node_modules/nsp` tree got bumped to `1.1.1` (the other stayed at the initial version).

I'm installing nsp as a local dependency. I see the docs instruct to install globally, but that is generally discouraged (and I try not to pollute my global space if it can be avoided). Does this make a difference?

What I don't understand is how that could happen, when the exact versions of all dependencies are supposed to be &ldquo;frozen&rdquo; in `npm-shrinkwrap.json`&hellip;
